src/frontend/src/lib/components/ui/ButtonMenu.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+	export let label: string;
+	export let ariaLabel: string | undefined = undefined;
+</script>
+
+<button
+	class={`flex gap-2 items-center no-underline hover:text-blue active:text-blue`}
+	aria-label={ariaLabel ?? label}
+	on:click
+>
+	<slot name="before-label"></slot>
+	{label}
```

Comment:
Sorry did not noticed in the first review, why not just `<slot />` here?

Then

```
<ButtonMenu label={$i18n.tokens.manage.text.title} on:click={modalStore.openIcManageTokens} />
```

becomes

```
<ButtonMenu on:click={modalStore.openIcManageTokens} >
   {$i18n.tokens.manage.text.title} 
</ButtonMenu>
```

It's closer to the rest of the code base. WDYT? Or rather like to pass it as prop?

Reply:
So, at first actually I didn't even put the slot, because I prefer to pass the label as prop, instead of component in slot.
However, there is one usage that requires a slot

```svelte
// old code
<button
	class="flex gap-2 items-center no-underline hover:text-blue active:text-blue"
	aria-label={$i18n.navigation.alt.more_settings}
>
	<IconSettings />
	{$i18n.settings.text.title}
</button>
```

and I thought this could be a nice solution, since most usage do not require additional components